admin-ui/app/company/index.tsx

```diff
@@ -194,15 +224,15 @@ const HomeComponent = ({ items }: HomeComponentProps) => {
               </Button>
             </DialogActions>
           </Dialog>
-          <Tooltip title='Export'>
-          <IconButton
-            aria-controls={Open ? "basic-menu" : undefined}
-            aria-haspopup="true"
-            aria-expanded={Open ? "true" : undefined}
-            onClick={handleClickData}
-          >
-            <FileDownloadOutlinedIcon />
-          </IconButton>
+          <Tooltip title="Export">
+            <IconButton
+              aria-controls={Open ? "basic-menu" : undefined}
```

Comment:
you dont need undefine and why do you need else .. and why variable is Open, follow the naming convention